core/internals.h

```diff
@@ -279,7 +279,7 @@ int uri_toString(const lwm2m_uri_t * uriP, uint8_t * buffer, size_t bufferLen, u
 
 // defined in objects.c
 uint8_t object_readData(lwm2m_context_t * contextP, lwm2m_uri_t * uriP, int * sizeP, lwm2m_data_t ** dataP);
-uint8_t object_read(lwm2m_context_t * contextP, lwm2m_uri_t * uriP, lwm2m_media_type_t * formatP, uint8_t ** bufferP, size_t * lengthP);
+uint8_t object_read(lwm2m_context_t * contextP, lwm2m_uri_t * uriP, uint8_t acceptNum, const uint16_t * accept, lwm2m_media_type_t * formatP, uint8_t ** bufferP, size_t * lengthP);
```

Comment:
Consistency: Everywhere else, the pointer to the array comes first, then the number elements it contains. I'd swap the arguments.

Reply:
Changed.